### ISSUE TITLE
Form/Input propTypes "type" should include more types

### ISSUE THREAD

**Issue by CodeDraken** [couds/react-bulma-components#78]
`type: PropTypes.oneOf(['text', 'email', 'tel', 'password', 'number', 'search'])`

Shouldn't this include color, file, and date as well?

**Comment by couds** [comment]
Hi @CodeDraken I think color and date could be, I'm planning to add the File input as another component because the behavior of its a little different (It is also another component on Bulma).

If you can fill a PR adding the other two types would be great. 

Thanks

**Comment by CodeDraken** [comment]
Yeah, I'll do a PR. Did you just want the types added or components for them?

**Comment by couds** [comment]
Can you add color and date to the type prop. File will be added as another component. Thanks
